src/test/integration/checkResponseCode.py

```diff
@@ -0,0 +1,65 @@
+import json
+import pycurl
+import urllib
+import StringIO
+
+def getAccessToken(url, client_id, client_secret):
+    post_data = "client_id=" + client_id + "&client_secret=" + client_secret + "&grant_type=client_credentials"
+
+    storage = StringIO.StringIO()
+    curl    = pycurl.Curl()
+
+    # Set options
+    curl.setopt(pycurl.URL, url)                     # CURLOPT_URL in PHP
+    curl.setopt(pycurl.POST, 1)                      # CURLOPT_POST in PHP
+    curl.setopt(pycurl.POSTFIELDS, post_data)        # CURLOPT_POSTFIELDS in PHP
+    curl.setopt(pycurl.WRITEFUNCTION, storage.write) # CURLOPT_RETURNTRANSFER in PHP
+
+    # Send the request and save response
+    curl.perform()
+
+    # Close request to clear up some resources
+    curl.close()
+
+    response = storage.getvalue()
+    return json.loads(response)
+
+def getResponse(url, access_token, params):
+    query_params = urllib.urlencode(params)
+    api_call_url = url + "?" + query_params;
+    headers      = ['Authorization: Bearer '+ access_token]
+
+    storage = StringIO.StringIO()
+    curl    = pycurl.Curl()
+
+    # Set options
+    curl.setopt(pycurl.URL, api_call_url)
+    curl.setopt(pycurl.HTTPHEADER, headers) # CURLOPT_HTTPHEADER in PHP
+    curl.setopt(pycurl.WRITEFUNCTION, storage.write)
+
+    curl.perform()
+    response_code = curl.getinfo(pycurl.HTTP_CODE)
+    curl.close()
+
+    response = storage.getvalue()
+    return response_code
+
+def checkResponseCode(response_code):
```

Comment:
Hmm, your code does bring up a good point that I don't have an answer for yet. Should our integration tests go through the API gateway or not?